idutils/__init__.py

```diff
@@ -384,6 +384,10 @@
 )
 """See https://www.wikidata.org/wiki/Property:P214."""
 
+email_regexp = re.compile(r"\S+@(\S+\.)+\S+")
+
+sha1_regexp = re.compile(r"^[a-zA-Z0-9]{40}$")
+
```

Comment:
very minor: we can place the https://www.w3.org/TR/annotation-model/#agents link here as comment to provide contextual information. All other regexp have a URL reference to their original id definition to provide that context/definition.

Reply:
Done.